lib/msf/core/auxiliary/report.rb

```diff
@@ -12,6 +12,38 @@ module Auxiliary::Report
 
   optionally_include_metasploit_credential_creation
 
+  def create_cracked_credential(opts={})
+    begin
+      super(opts)
+    rescue NoMethodError => e
```

Comment:
i'll try it with just checking if the db is active, i wasn't convinced that would work properly in the case of installing without the db support libraries, but i didn't actually try it for whatever reason. if it works, i'll update